action.yml

```diff
@@ -100,7 +100,10 @@ runs:
 
         # Clean previous image, if any
         IS=${{ steps.vars.outputs.imageStream }}
```

Comment:
wrapping it with double quotes would be better I believe , thoughts?
`IS="${{ steps.vars.outputs.imageStream }}"`

Reply:
Good catch and added!  :)